e2e/src/pages/commentspanel.po.ts

```diff
@@ -0,0 +1,103 @@
+import {browser, by, element, ElementArrayFinder, ElementFinder,} from 'protractor';
+import {AppPage} from './app.po';
+import {By} from '@angular/platform-browser';
+import {expect} from 'chai';
+
+
+export class CommentsPanelPage extends AppPage {
+  //commentsToggleButton : ElementFinder = element(by.id('toggle-comments-panel'));
+
+  commentsToggleButton : ElementFinder      = element(by.xpath("//button[@id=\'toggle-comments-panel\' and @title=\'Toggle Comments Panel\']"));
```

Comment:
why do we need to check for the title here?

Reply:
removed the check for title .